### ISSUE TITLE
Add the "Speakers Wanted" board link to the "Get Involved" page on the Techlahoma website

### ISSUE THREAD

**Issue by kacollins** [techlahoma/help-wanted#96]
Add "Speak" as an option after "Donate" and "Volunteer" on the [Get Involved](https://www.techlahoma.org/get-involved) page. An "I want to speak" button should link to the [Speakers Wanted Board](https://speakers-wanted.techlahoma.org).

Committee: Website and Social Media
Slack: [#techlahoma-website](https://techlahoma.slack.com/messages/techlahoma-website)

**Comment by stephaniepurvis** [comment]
Should there be a description to go under the "Speak" title?

**Comment by kacollins** [comment]
Yep! Would you be able to write something? Feel free to ask for help in the #techlahoma-volunteers channel.

**Comment by randypitcherii** [comment]
Do you still need help on this? If so, I have a few questions:
- which repo holds your website
- anything special I need to know about submitting a PR with this change to your repo?
- is deployment automated for this site (will a push to master result in the site updating itself?)

@stephaniepurvis have you already started working on this? If so, I can find a different task to jump in on.

Thanks! I'm looking forward to jumping in.

**Comment by kacollins** [comment]
Unfortunately, the site is on Squarespace, not GitHub. We can get you edit access if Stephanie isn't taking this one.
